Sources/Testing/Issues/Issue+Recording.swift

```diff
@@ -54,6 +54,15 @@ extension Issue {
     return issue.record(configuration: configuration)
   }
 
+  /// Record this issue by wrapping it in an ``Event`` and passing it to the
+  /// current event handler.
+  ///
+  /// - Returns: The issue that was recorded (`self` or a modified copy of it.)
```

Comment:
We should make sure the API documentation explains _what_ might be modified (the SPI documentation is, and this is a technical term here, "lazy".)